tests/fuzz/fuzz_x509crt.c

```diff
@@ -0,0 +1,22 @@
+#include <stdint.h>
+#include "mbedtls/x509_crt.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+#ifdef MBEDTLS_X509_CRT_PARSE_C
```

Comment:
The house style is to always use `#if defined(…)` and not `#ifdef` except for header double-inclusion guards. That's just a matter of style, it doesn't break any tools that we use, so as far as I'm concerned this is not a blocker.

Reply:
Tell me if you want me to change this